Makefile.am

```diff
@@ -97,8 +98,7 @@ _sockets_files = \
 	prov/sockets/src/sock_rma.c \
 	prov/sockets/src/sock_atomic.c \
 	prov/sockets/src/sock_trigger.c \
-	prov/sockets/src/sock_util.h \
-	prov/sockets/src/indexer.c
+	prov/sockets/src/sock_util.h
```

Comment:
Does the socket provider still include the file when built as a dynamic library?